diff --git a/node_modules/react-native-fast-squircle/ios/FastSquircleBorderDrawing.m b/node_modules/react-native-fast-squircle/ios/FastSquircleBorderDrawing.m
index 4621b25..b60690c 100644
--- a/node_modules/react-native-fast-squircle/ios/FastSquircleBorderDrawing.m
+++ b/node_modules/react-native-fast-squircle/ios/FastSquircleBorderDrawing.m
@@ -7,7 +7,7 @@
 
 #import "FastSquircleBorderDrawing.h"
 #import <React/RCTUtils.h>
-#import "FastSquircle-Swift.h"
+#import <FastSquircle/FastSquircle-Swift.h>
 
 static const CGFloat RCTViewBorderThreshold = 0.001;
 
diff --git a/node_modules/react-native-fast-squircle/ios/FastSquircleBoxShadow_81orMore.mm b/node_modules/react-native-fast-squircle/ios/FastSquircleBoxShadow_81orMore.mm
index e2c39e1..4406e99 100644
--- a/node_modules/react-native-fast-squircle/ios/FastSquircleBoxShadow_81orMore.mm
+++ b/node_modules/react-native-fast-squircle/ios/FastSquircleBoxShadow_81orMore.mm
@@ -11,7 +11,7 @@
 
 #import "FastSquircleBoxShadow_81orMore.h"
 #import <React/RCTConversions.h>
-#import "FastSquircle-Swift.h"
+#import <FastSquircle/FastSquircle-Swift.h>
 
 static CGFloat adjustedCornerRadius(CGFloat cornerRadius, CGFloat spreadDistance)
 {
diff --git a/node_modules/react-native-fast-squircle/ios/FastSquircleView.mm b/node_modules/react-native-fast-squircle/ios/FastSquircleView.mm
index 0a6ee61..a50dd46 100644
--- a/node_modules/react-native-fast-squircle/ios/FastSquircleView.mm
+++ b/node_modules/react-native-fast-squircle/ios/FastSquircleView.mm
@@ -12,7 +12,7 @@
 #import <React/RCTBorderDrawing.h>
 #import <React/RCTConversions.h>
 #import "FastSquircleBorderDrawing.h"
-#import "FastSquircle-Swift.h"
+#import <FastSquircle/FastSquircle-Swift.h>
 #if REACT_NATIVE_VERSION_MAJOR > 0 || REACT_NATIVE_VERSION_MINOR >= 81
 #import "FastSquircleBoxShadow_81orMore.h"
 #else
diff --git a/node_modules/react-native-fast-squircle/ios/SquirclePathGenerator.swift b/node_modules/react-native-fast-squircle/ios/SquirclePathGenerator.swift
index 72fc761..df8ee1d 100644
--- a/node_modules/react-native-fast-squircle/ios/SquirclePathGenerator.swift
+++ b/node_modules/react-native-fast-squircle/ios/SquirclePathGenerator.swift
@@ -105,7 +105,7 @@ func getSquirclePath(params: SquircleParams) -> UIBezierPath {
       cornerRadius: min(roundingAndSmoothingBudget, bottomRightCornerRadius),
       cornerSmoothing: cornerSmoothing,
       roundingAndSmoothingBudget: roundingAndSmoothingBudget
-    )),
+    ))
   )
 }
 
